<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mnue</class>
 <widget class="QWidget" name="Mnue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="userName">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>昵称</string>
   </property>
  </widget>
  <widget class="QLabel" name="motto">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>人生格言</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: none;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: none;</string>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTreeView" name="FriendList">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>350</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: none;</string>
   </property>
  </widget>
  <widget class="QLabel" name="topBg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>351</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSetting">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>670</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>670</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ImageLabel" name="profilePicture">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>topBg</zorder>
  <zorder>userName</zorder>
  <zorder>motto</zorder>
  <zorder>search</zorder>
  <zorder>pushButton</zorder>
  <zorder>FriendList</zorder>
  <zorder>buttonSetting</zorder>
  <zorder>profilePicture</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageLabel</class>
   <extends>QLabel</extends>
   <header>imagelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
